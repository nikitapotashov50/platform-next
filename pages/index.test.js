import test from 'ava'
import render from 'react-test-renderer'

import Index from '.'

test('Index page', t => {
  const tree = render.create(<Index />).toJSON()
  t.snapshot(tree)
})
