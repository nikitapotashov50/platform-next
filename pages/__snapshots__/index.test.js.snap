// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld component 1`] = `
<div>
  Index page!
</div>
`;

exports[`Index page 1`] = `
<div>
  Index page!
</div>
`;
